packages/itwin/geo-tools/src/AddressProvider.ts

```diff
@@ -64,11 +59,12 @@ export class BingAddressProvider implements AddressProvider {
    * Latitudes(Y) values are kept between -PI and +PI while
    */
   public async getAddresses(query: string, viewLatLongBBox: Range2d): Promise<AddressData[]> {
-    const requestUrl = this.getUrl(query, viewLatLongBBox);
-    const requestOptions: RequestOptions = { method: "GET", responseType: "json" };
+    const url = this.getUrl(query, viewLatLongBBox);
+
     try {
-      const locationResponse: Response = await request(this._requestContext, requestUrl, requestOptions);
-      const value = locationResponse.body.resourceSets[0].resources[0].value;
+      const response = await fetch(url, {method: "GET"});
+      const json: any = await response.json();
```

Comment:
should we type this? add some error handling on the line below this?

Reply:
@aruniverse

Reply:
That makes sense re typing the large response format. Could you add a link to the doc you provided in the code instead then?